### ISSUE TITLE
Monoprice Mini Delta V2

### ISSUE THREAD

**Issue by Xowap** [OctoPrint/OctoPrint-MalyanConnectionFix#8]
Hi all!

I've just acquired a Monoprice Mini Delta V2, kinda expecting that things weren't going to be smooth and... Well they are not.

In a nutshell, from the Octoprint point of view, I never receive any reply to anything so the connection is completely dead, whatever the baud rate.

An interesting thing though is that without the plugin, when I connect the fans just stop running (wtf?) and the printer restarts after I disconnect.

With the plugin, sometimes the fans don't stop running and the printer restarts _anyways_ after I disconnect.

This makes not a single sense at all.

I'm quite surprised at the print quality from this printer when the software is clearly sub-standard...

Anyways, my question is: how do I debug this? What can I do to make the plugin compatible?

Thanks!

**Comment by foosel** [comment]
Hi @Xowap

I fear I won't be much help in that regard, and I'm not sure anyone else even actively monitors this issue tracker. Your chances to find other people with the same printer and possible solutions to your problem are way bigger on the [forums](https://community.octoprint.org) or [the Discord server](https://discord.octoprint.org).

The only reason I was able to implement this plugin was because some forum users reported observations on how they were able to make things work (double connect), and I then turned this into a plugin. I actually have since acquired one of the affected printers myself, but it's sadly not a Monoprice Mini Delta **V2**, so I can't assist in troubleshooting and also have no idea where to even begin - I have no idea how people discovered the original workaround in the first place, as you said, it all makes little sense in the first place.

However, IF there exists a workaround, I'm happy to see if it can be either added to this or implemented as its own plugin, or assist someone else doing so. Until then however, best seek your luck in getting to the bottom of this in the aforementioned forum or discord server. I'll close this as its not an immediate actionable task for me as the maintainer, and there are better places for discussions, community exchange and joint brainstorming.
